libr/util/sys.c

```diff
@@ -460,6 +460,37 @@ R_API int r_sys_chdir(const char *s) {
 	return r_sandbox_chdir (s)==0;
 }
 
+R_API bool r_sys_aslr(int val) {
+	bool ret = true;
+#if __linux__
+	static const char *rva = "/proc/sys/kernel/randomize_va_space";
+	char buf[2] = {0};
```

Comment:
the string you are trying to set in here is 3 byte long, not 2, this is overflowing

Reply:
Originally we write two bytes to the file but 3 works too ...